### PR TITLE
Added type definition for react-flexr

### PR DESCRIPTION
**Pull request by pushplay**
case 1. Add a new type definition.
- [x] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [x] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [x] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

### CODE REVIEW

File: react-flexr/react-flexr-tests.tsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
+/// <reference path="react-flexr.d.ts"/>
+/// <reference path="../react/react-dom.d.ts"/>
+
+import * as React from 'react';
+import * as ReactDOMServer from "react-dom/server";
```
Comment (zhengbli) [inline react-flexr/react-flexr-tests.tsx]: Use `'` or `"` consistently in the file, don't mix them.

File: react-flexr/react-flexr.d.ts [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+        /**
+         * Takes a fraction. e.g. 1/6
+         */
+        size?: string | number;
+
+        /**
+         * Like size, but only for palm devices.
+         * Accepts 'hidden' as well.
+         */
+        palm?: string | number;
```
Comment (zhengbli) [inline react-flexr/react-flexr.d.ts]: You can make it `number | "hidden"` to be more specific

Comment (pushplay) [inline react-flexr/react-flexr.d.ts]: That's not quite right because `like size` it can accept other strings.  eg: '1/6'.

**Comment by pushplay** [comment]
I fixed the quotes issue.  It should be good to merge now.

**Comment by zhengbli** [comment]
:+1:

**Pull request merged**